python/desc/sims/GCRCatSimInterface/simCat_DC2sndb.py

```diff
@@ -0,0 +1,170 @@
+"""
```

Comment:
Adding this class (and the jupyter notebooks that use it) adds a dependence on `snsims`, which brings with it a dependence on `esheldon/fitsio` and `OpSimSummary`.  Do we want `sims_GCRCatSimInterface` to depend on those packages, or, given that the infrastructure being added here just reads in the supernova populations from a csv file, should another repository be created that contains the code to actually generate those csv files, while the repository just takes it on faith that the csv files being supplied are correct?

Reply:
Scott: That is a mistake. I don't want to keep this in this repository. It is not going to be used in production

Reply:
@rbiswas4 I thought you were going to remove this file (and, presumably, the notebooks that depend on it)?

Reply:
Yes. I am going to run `git rm` . If you have a suggestion of where to keep this that is good. Otherwise I will put them in my repository.

Reply:
It seems like these scripts are still in a state where only you can run them (i.e. they depend on some of your other repositories).  In that case, I would say: keep them in your own personal repository for now.  We can work on making these scripts publicly usable at another time.